src/components/CreateResolution/UploadResolution.vue

```diff
@@ -0,0 +1,374 @@
+                      <span v-if="partialItem.type === ItemTypes.PLACEHOLDER
+                                && partialItem.value === PLACEHOLDER_LEGAL_NAME_INLINE">
+                        {{getBusinessLegalName}}
+                      </span>
+                      <div v-if="partialItem.type === ItemTypes.PLACEHOLDER
+                                && partialItem.value === PLACEHOLDER_LEGAL_NAME">
+                        <strong>{{getBusinessLegalName}}</strong>
+                      </div>
+                    </span>
+                  </v-col>
+                </v-row>
+              </li>
+            </ul>
+          </v-form>
+        </v-card>
+      </div>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Mixins, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+
+// Interfaces
+import {
+  ActionBindingIF,
+  CreateResolutionIF,
+  CreateResolutionResourceIF,
+  FormIF,
+  NameRequestDetailsIF,
+  ValidationDetailIF
+} from '@/interfaces'
+
+// Enums
+import { BulletListTypes, CorpTypeCd, ItemTypes, RouteNames } from '@/enums'
+
+// Mixins
+import { CommonMixin, EntityFilterMixin } from '@/mixins'
+
+@Component({})
+export default class UploadResolution extends Mixins(CommonMixin, EntityFilterMixin) {
+  // Refs
+  $refs!: {
+    confirmResolutionChk: FormIF
+  }
+
+  private hasResolutionConfirmed = false
+  private resolutionConfirmed = false
+  private helpToggle = false
+  private PLACEHOLDER_LEGAL_NAME = 'legal_name'
+  private PLACEHOLDER_LEGAL_NAME_INLINE = 'legal_name_inline'
+
+  @Getter getShowErrors!: boolean
+  @Getter getNameRequestDetails!: NameRequestDetailsIF
+  @Getter getCreateResolutionResource!: CreateResolutionResourceIF
+  @Getter getCreateResolutionStep!: CreateResolutionIF
+  @Getter getValidateSteps!: boolean
+  @Getter getBusinessLegalName!: string
+
+  @Action setResolution!: ActionBindingIF
+  @Action setResolutionStepValidationDetail!: ActionBindingIF
+
+  // Enum for template
+  readonly RouteNames = RouteNames
+  readonly ItemTypes = ItemTypes
+  readonly CorpTypeCd = CorpTypeCd
+  readonly BulletListTypes = BulletListTypes
+
+  private get documentURL ():string {
+    const docUrl = sessionStorage.getItem('BASE_URL') +
+      this.getCreateResolutionResource.sampleFormSection.downloadDocPath
+    return docUrl
+  }
+
+  private previewImageSource ():string {
+    // Note: the image file path did not resolve correctly when using the require function directly.  In order
+    // to get the image path resolving correctly, needed to get the image context first and use that to build
+    // the final image image file path
+    const images = require.context('@/assets/images/', false, /\.png$/)
+    const previewImageSrc = images('./' + this.getCreateResolutionResource.sampleFormSection.previewImagePath)
+    return previewImageSrc
+  }
```

Comment:
Pretty cool!

Could this be simplified to `return images('./' + this.getCreateResolutionResource.sampleFormSection.previewImagePath)` ?

Reply:
i kind of do this on purpose as i like placing breakpoints on the following line.  i.e. `return previewImageSrc`  But I do think it's definitely not as elegant too.  Anyways, I updated it 😋

Reply:
^^ same for me

Reply:
😁 All good either way, thanks!